packages/select-text/controller/src/index.js

```diff
@@ -6,10 +6,29 @@ import defaults from './defaults';
 
 const log = debug('@pie-element:select-text:controller');
 
-const buildTokens = (tokens, evaluateMode) => {
-  tokens = tokens || [];
+const isAnswerSelected = (token, selectedToken) => {
+  return (
+    token.correct === true &&
```

Comment:
Please add sanity check on token, and remove { return (...) }. You can simply do => (...)